improver_tests/utilities/test_neighbourhood_tools.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2020 Met Office.
```

Comment:
There is a PR open to update all copyright dates to range 2017-2021. As this is a new file it won't be captured by that PR, so please update here. Thanks.
```suggestion
# (C) British Crown Copyright 2017-2021 Met Office.
```